source/command-monitoring/tests/find.yml

```diff
@@ -247,6 +248,76 @@ tests:
               nextBatch:
                 - { _id: 4, x: 44 }
           command_name: "getMore"
+  -
+    description: "A successful find event with a multiple getMore (>= 5.0)"
+    ignore_if_server_version_less_than: "5.0"
+    ignore_if_topology_type:
+      - "sharded"
+    operation:
+      name: "find"
+      arguments:
+        filter: { _id: { $gte: 1 }}
+        sort: { _id: 1 }
+        batchSize: {"$numberLong": "3"}
+        limit: {"$numberLong": "4"}
```

Comment:
Is it necessary to use `$numberLong`, or is this just copying what was done in other tests?

I notice we don't use it for all numerics (`filter` and `sort` above for instance).

Reply:
Yes, this was pure copy/paste here.